EU4toV2/Source/V2World/Pops/PopType.h

```diff
@@ -0,0 +1,22 @@
+#ifndef POP_TYPE_H
+#define POP_TYPE_H
+
+#include "newParser.h"
+#include "Pop.h"
+#include <map>
+
+namespace mappers
+{
+	class PopType : commonItems::parser
+	{
+	public:
+		PopType() = default;
+		PopType(std::istream& theStream);
+		std::map<std::string, Pop> getPopTypes() const { return popTypes; }
+
+	private:
+		std::map<std::string, Pop> popTypes;
```

Comment:
Isn't it possible to have multiple pops of a single type in a province? This would only allow for one of each type to be stored. Might want to use std::multimap instead.

Reply:
AFAIK no, and if it is, there is something broken. You can't/shouldn't be able to have two separate stacks of English protestant soldiers. within a single province - it would cause massive issues for recruitment. Do you have an example of this?

Reply:
Pardon my fault for not reading the code correctly. Yes, I filter by type only and we'll indeed have multiple. Will use multimap.